src/collection/views/CollectionEdit.tsx

```diff
@@ -361,10 +363,23 @@ const CollectionEdit: FunctionComponent<CollectionEditProps> = ({
 					label={t('collection/views/collection-edit___delen')}
 					disabled={hasUnsavedChanged()}
 					title={
-						!eq(currentCollection, initialCollection) ? 'U moet uw wijzigingen eerst opslaan' : ''
+						!eq(currentCollection, initialCollection)
+							? t('U moet uw wijzigingen eerst opslaan')
```

Comment:
Extract translation

Reply:
@andrych @benjaminnaesen 
discussion:
Gaan we verplichten dat vertalingen ge-extract worden bij iedere PR of eisen we enkel dat ze gemarkeerd zijn als translatable?

Dan geraken ze zowiezo wel ooit eens in de translation file en moeten we die "trage" extract translations niet runnen bij iedere PR?